### PR TITLE
Greenhouse no open positions

### PR DESCRIPTION
**Pull request by ChanyRosental**
This scrapers use greenhouse, and no open positions, so the test fails
What do we do in this case?

### CODE REVIEW

File: Scrapers/CompanyScrapers/CodefreshScraper.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class CodefreshScraper(GreenhouseScraper): 
+    url = "https://boards.greenhouse.io/codefresh"
+    name = 'Codefresh'
+
+    def scrape(self):
+        super().scrape()
+
+CodefreshScraper().check_self()
```
Comment (snirsh) [inline Scrapers/CompanyScrapers/CodefreshScraper.py:12]: can be removed

File: Scrapers/CompanyScrapers/NomadHealthScraper.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+from ScrapingTools.OutsourceTools.Greenhouse.GreenhouseScraper import GreenhouseScraper
+
+
+class NomadHealthScraper(GreenhouseScraper):
+    url = "https://boards.greenhouse.io/nomadhealth"
+    name = 'Nomad Health'
+
+    def scrape(self):
```
Comment (snirsh) [inline Scrapers/CompanyScrapers/NomadHealthScraper.py:9]: we can remove all the supers

**Review by snirsh — DISMISSED** [review]

File: Scrapers/CompanyScrapers/CodefreshScraper.py (hunk 1 above)
Comment (snirsh) [reply Scrapers/CompanyScrapers/CodefreshScraper.py:12]:
```suggestion
```

**Review by snirsh — COMMENTED** [review]
remove supers

File: Scrapers/CompanyScrapers/VaretoScraper.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+from ScrapingTools.OutsourceTools.Greenhouse.GreenhouseScraper import GreenhouseScraper
+
+
+class VaretoScraper(GreenhouseScraper): 
+    url = "https://boards.greenhouse.io/vareto"
```
Comment (snirsh) [inline Scrapers/CompanyScrapers/VaretoScraper.py:6]: `https://api.greenhouse.io/v1/boards/vareto/embed/jobs`
Check this, we can make them all over their requests API, check my other comment for the appropriate code :)

File: ScrapingTools/OutsourceTools/Greenhouse/GreenhouseScraper.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from Scrapers.Scraper import *
+
+
+class GreenhouseScraper(Scraper):
+    base_url = 'https://boards.greenhouse.io/'
+    url = 'https://boards.greenhouse.io/'
+    name = 'Greenhouse'
+
+    def scrape(self):
+        soup = self.scraping_unit(self.url)
```
Comment (snirsh) [inline ScrapingTools/OutsourceTools/Greenhouse/GreenhouseScraper.py:10]:
```suggestion
        response = requests.get(self.url)

        response_json = response.json()

        for job in response_json.get('jobs'):
            self.positions.append(
                self.Position(
                    title=job['title'].strip(),
                    link=job['absolute_url'].strip(),
                    location=job['location']['name'].strip()
                )
            )
```

**Review by snirsh — CHANGES_REQUESTED** [review]
LMK WDYT

File: Scrapers/CompanyScrapers/VaretoScraper.py (hunk 3 above)
Comment (noazinger) [reply Scrapers/CompanyScrapers/VaretoScraper.py]: we tried to modify the code as you mentioned 
it works when the website has no open position but if it has it gives the following error:response_json = response.json() requests.exceptions.JSONDecodeError
what was the problem with our code, can't we use it

**Review by snirsh — DISMISSED** [review]

File: Scrapers/CompanyScrapers/VaretoScraper.py (hunk 3 above)
Comment (snirsh) [reply Scrapers/CompanyScrapers/VaretoScraper.py]: Try adding headers to the request

**Comment by snirsh** [comment]
Please add the option that we talked about with using two different scrapers and if one fails you run the other, for example:
```
def scrape_with_slug(self):
...
def scrape_without_slug(self):
...

def scrape(self):
    try:
        self.scrape_with_slug()
    catch e:
        self.scrape_without_slug()
```

**Comment by snirsh** [comment]
Remove all the deleted files here, i dont understsand what it is

**Review by snirsh — CHANGES_REQUESTED** [review]
Check my comments please

**Review by snirsh — APPROVED** [review]

**Pull request merged**